plugins/processors/k8sdecorator/stores/utils.go

```diff
@@ -51,42 +53,52 @@ func parseDeploymentFromReplicaSet(name string) string {
 		return ""
 	}
 
-	if !stringInRuneset(suffix, kubeAllowedStringAlphaNums) {
+	if containUnexpectedRuneSet(suffix,deploymentUnexpectedRegExp) {
```

Comment:
Nitpick: spacing between the arguments
```suggestion
	if containUnexpectedRuneSet(suffix, deploymentUnexpectedRegExp) {
```